### PR TITLE
Fix batch insert and add batchPutAllItems

### PR DESCRIPTION
**Pull request by dieend**

### CODE REVIEW

File: src/Command.php [hunk 1]
```diff
@@ -161,11 +161,50 @@ public function putItem($table, array $value, array $options = [])
...
+     * @return static[]
+     */
+    public function batchPutAllItems($table, array $values, array $options = [])
+    {
+        $batches = array_chunk($values, 25);
+        $commandOriginal = $this;
+        return array_map(
+            function ($batch) use ($commandOriginal, $table, $options) {
+                list($name, $argument) = $this->db->getQueryBuilder()->batchPutItem($table, $batch, $options);
+                $command = clone $commandOriginal;
```
Comment (petrabarus) [inline src/Command.php]: we can't do `$command = clone $this`? looks okay, though, if we can't.

Comment (dieend) [inline src/Command.php]: What we can't is passing `$this` into function use.

Comment (petrabarus) [inline src/Command.php]: But we can use `$this` in the statement above.

Comment (dieend) [inline src/Command.php]: `$this` will point to `Command` object? Sorry, my mind clouded with JavaScript where the `$this` points to the closure @_@

Comment (petrabarus) [inline src/Command.php]: Yup

Comment (dieend) [inline src/Command.php]: Fixed.

File: src/Command.php [hunk 2]
```diff
@@ -161,11 +161,49 @@ public function putItem($table, array $value, array $options = [])
...
+     * @param array  $values  The values to input.
+     * @param array  $options Additional options to the request argument.
+     * @return static[]
+     */
+    public function batchPutAllItems($table, array $values, array $options = [])
+    {
+        $batches = array_chunk($values, 25);
+        return array_map(
+            function ($batch) use ($table, $options) {
+                list($name, $argument) = $this->db->getQueryBuilder()->batchPutItem($table, $batch, $options);
```
Comment (petrabarus) [inline src/Command.php]: Is it the same if like this

```
$command = clone $this;
return $command->batchPutItem($table, $batch, $options)
```

Comment (dieend) [inline src/Command.php]: Yep. It's equal. Updated.

**Pull request merged**